android/app/build.gradle

```diff
@@ -78,7 +78,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    enableHermes: false,  // clean and rebuild if changing
+    enableHermes: true,  // clean and rebuild if changing
```

Comment:
do you wanna clean and rebuild every single time?
what was not working?

Reply:
This is in response to this error:
Error: Requiring module "node_modules/react-native-reanimated/src/reanimated2/index.js", which threw an exception: TypeError: null is not an object (evaluating 'InnerNativeModule.installCoreFunctions')

Solution from this github post:
software-mansion/react-native-reanimated#846 (comment)

Both this addition and the one in MainApplication.java seem to address the same issue.